Automattic/wp-calypso
# Atomic: Move PHP Version Card to Redux

#### Changes proposed in this Pull Request

Move data handling to `data-layer` middleware.

#### Testing instructions

- Go to the page with active Hosting Panel
- Try to update PHP version
- Make sure it's correctly updated, and correct notices displayed
- Make sure it works when updates fail (force back-end to error)
